app/assets/stylesheets/content/_diff.sass

```diff
@@ -45,3 +45,4 @@ ins
   background-color: #f6f6f6
   color: #505050
   border: 1px solid #e4e4e4
+  line-height: 1.75rem
```

Comment:
Why did you choose such a large value?

`line-height: normal` seems to be working just fine, and does not make the line spacing unusually large.

Don't get me wrong. It seems like oyu identified the correct problem, because the `line-height` was way too low. I just don't think that we need to make it that large.